lms/djangoapps/instructor/tests/test_ecommerce.py

```diff
@@ -53,6 +66,24 @@ def test_pass_e_commerce_tab_in_instructor_dashboard(self):
         # Coupons should show up for White Label sites with priced honor modes.
         self.assertIn('Coupon Code List', response.content)
 
+    @mock.patch("openedx.core.djangoapps.site_configuration.helpers.get_value", fake_get_value)
```

Comment:
Thanks, @douglashall can you guide how can I take the advantage from [site_configuration_other](https://github.com/edx/edx-platform/blob/d1a3af9460e0381aeb69a42632902370eb7bd505/openedx/core/djangoapps/site_configuration/tests/mixins.py#L27) in my test cases ? 
Or need to add my new value in `show_ecommerce_reports` under [site_configurations](https://github.com/edx/edx-platform/blob/d1a3af9460e0381aeb69a42632902370eb7bd505/openedx/core/djangoapps/site_configuration/tests/mixins.py#L15) ? 
e.g. 
```
        self.site = SiteFactory.create()
        self.site_configuration = SiteConfigurationFactory.create(
            site=self.site,
            values={
                "SITE_NAME": self.site.domain,
                "course_org_filter": "fakeX",
                "show_ecommerce_reports": True,
            }
        )
```